playmusicexporter/src/main/res/values/strings.xml

```diff
@@ -146,4 +146,5 @@
     <string name="file_handling_category">File Handling</string>
     <string name="overwrite_summary">Always overwrite the exported file, even if it already exists.</string>
     <string name="overwrite_title">Overwrite Existing Files</string>
+	<string name="action_refesh">Refresh</string>
```

Comment:
Hmm, Android Studio did that (added via the UI), I wonder if it's the eternal tabs vs spaces epic battle :-)